src/EditorFeatures/Core.Wpf/InlineParameterNameHints/InlineParamNameHintsTag.cs

```diff
@@ -0,0 +1,45 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Windows;
+using System.Windows.Controls;
+using Microsoft.VisualStudio.Text;
+using Microsoft.VisualStudio.Text.Editor;
+
+namespace Microsoft.CodeAnalysis.Editor.InlineParameterNameHints
+{
+    /// <summary>
+    /// This is the tag which implements the IntraTextAdornmentTag and is meant to create the UIElements that get shown
+    /// in the editor
+    /// </summary>
+    internal class InlineParamNameHintsTag : IntraTextAdornmentTag
```

Comment:
```suggestion
    internal class InlineParameterNameHintsTag : IntraTextAdornmentTag
```

And also rename the file.